spotify_abs_cls.py

```diff
@@ -0,0 +1,16 @@
+import utils
+
+
+class SpotifuHandler(object):
+    """
+    Superclass for spotify fun
+    """
+    def __init__(self, spotify_yaml_path):
```

Comment:
Now that we have a super-class (nice!), I think that most utils can move to the superclass instead of a separate file